src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,210 @@
+package duke;
+
+import duke.command.CommandList;
+import duke.task.Task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class Duke {
+    private static final int FOUR = 4;
+    private static final int FIVE = 5;
+    private static final int SIX = 6;;
+    private static final int EIGHT = 8;
+    private static final int NINE = 9;
```

Comment:
These string constants to refer to an integer may seem redundant and self-explanatory. 
If you are working to replace them with magic numbers I would suggest more meaningful names.